src/mapping/impl/BasisFunctions.hpp

```diff
@@ -149,12 +149,12 @@ class Gaussian : public CompactSupportBase {
       return std::exp(-std::pow(_shape * radius, 2.0)) - _deltaY;
   }
 
+  /// Below that value the function is supposed to be zero. Defines the support radius if not explicitely given
+  static constexpr double cutoffThreshold = 1e-9;
+
```

Comment:
Mmh, static. Couldn't we treat it like `NUMERICAL_ZERO_DIFFERENCE` in `math/differences.hpp`?

Reply:
Note that this is a `static` class member and not a static variable, which is something different. Also, the `cutoffThreshold` is something different then `NUMERICAL_ZERO_DIFFERENCE`. I still think that a member variable is the way to go here, but I'm open for discussions.

Reply:
Shouldn't this be configurable by the user?

Reply:
The cutoff threshold is and was constant just as before. Making it configurable might be desirable, but not the purpose of this PR.

Reply:
I opened an issue https://github.com/precice/precice/issues/1265